bims/static/js/views/search.js

```diff
@@ -132,6 +133,12 @@ define([
                 this.search(searchValue);
             }
         },
+        clearSearch: function () {
+            $('#search').val('');
```

Comment:
hi @ann26 
it should also clear the filters too

Reply:
ok